app/src/main/java/com/sopt/now/MainActivity.kt

```diff
@@ -11,23 +16,59 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
-        val userData = intent.getParcelableExtra<UserDataInput>(LoginActivity.INTENT_USER_DATA)
-
-        val signId = userData?.getUserSignUpId()
-        val signPw = userData?.getUserSignUpPw()
-        val signNickname = userData?.getUserSignUpNickName()
-        val signMbti = userData?.getUserSignUpMbti()
-
-        with(binding) {
-            tvUserId.text = "ID: $signId"
-            tvUserPw.text = "Password: $signPw"
-            tvUserName.text = "Name: $signNickname"
-            tvUserMbti.text = "MBTI: $signMbti"
+
+
+        val currentFragment = supportFragmentManager.findFragmentById(binding.fcvHome.id)
+        if (currentFragment == null) {
+            supportFragmentManager.beginTransaction()
+                .add(binding.fcvHome.id, HomeFragment())
+                .commit()
         }
+
+        clickBottomNavigation()
+
+    }
+
+
+    private fun clickBottomNavigation() {
+        binding.bnvHome.setOnItemSelectedListener {
+            when (it.itemId) {
+                R.id.menu_home -> {
+                    replaceFragment(HomeFragment())
+                    true
+                }
+
+                R.id.menu_search -> {
+                    replaceFragment(SearchFragment())
+                    true
+                }
+
+                R.id.menu_mypage -> {
+                    val userData =
+                        intent.getParcelableExtra<UserDataInput>(LoginActivity.INTENT_USER_DATA)
+                    val myPageFragment = MyPageFragment()
+                    val bundle = Bundle()
+                    bundle.putParcelable("userData", userData)
+                    myPageFragment.arguments = bundle
+                    replaceFragment(myPageFragment)
```

Comment:
scope function을 사용하면 더 깔끔하게 작성할 수 있을 것 같아요! 
```suggestion
                   replaceFragment(MyPageFragment().apply {
                        arguments = Bundle().apply {
                            putParcelable("userData", userData)
                        }
                    })
```
이런 식으로요!

Reply:
미쳤다.. scope의 여신.. 신.. 감사합니다.